src/theory/quantifiers/ieval/quant_info.h

```diff
@@ -0,0 +1,126 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Info per quantified formula in instantiation evaluator.
+ */
+
+#include "cvc5_private.h"
+
+#ifndef CVC5__THEORY__QUANTIFIERS__IEVAL__QUANT_INFO_H
+#define CVC5__THEORY__QUANTIFIERS__IEVAL__QUANT_INFO_H
+
+#include <map>
+
+#include "context/cdo.h"
+#include "expr/node.h"
+
+namespace cvc5::internal {
+namespace theory {
+namespace quantifiers {
+namespace ieval {
+
+/**
+ * Stores all static information for a quantified formula. Also maintains
+ * context-dependent state information for the quantified formula when using
+ * the instantiation evaluator.
+ */
+class QuantInfo
+{
+ public:
+  QuantInfo(context::Context* c);
+  /**
+   * Initialize, called once.
+   */
+  void initialize(TNode q, Node body);
+  //-------------------------- static information
+  /**
+   * Get the constraints, which maps pattern terms to node corresponding to
```

Comment:
I think you could rephrase to clarify. I think "node" should be Boolean and the "their constraints" are whether they must hold or not? Moreover, clarify that this is supposed to only cover patterns which are predicates.